certbot/cli.py

```diff
@@ -398,11 +398,6 @@ def _get_help_string(self, action):
         "usage": "\n\n  certbot install --cert-path /path/to/fullchain.pem "
         " --key-path /path/to/private-key [options]\n\n"
     }),
-    ("config_changes", {
-        "short": "Show changes that Certbot has made to server configurations",
-        "opts": "Options for viewing configuration changes",
-        "usage": "\n\n  certbot config_changes [options]\n\n"
-    }),
     ("rollback", {
         "short": "Roll back server conf changes made during certificate installation",
         "opts": "Options for rolling back server configuration changes",
```

Comment:
I can't comment on it directly, but I think we should also delete the line https://github.com/certbot/certbot/pull/7469/files#diff-98e51bbf59cb44db5dc06a7c5a002983R445.